app/views/photos/home.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="container-fluid" style="padding-top: 20px; padding-left: 30px">
+	<br>
+	<h1 class="welcome">Welcome to the WICI Website!</h1><br><br>
+
+<div class="container-fluid">
+  <div class="row">
+    <% @photos.each do |photo| %>
+      <div class="col-4 home-image" onmouseover="showCaption(this)" onmouseleave="hideCaption(this)">
+        <%= image_tag photo.image, class: "image-size" %>
+        <div class="caption-holder">
+          <%= photo.description %>
+        </div>
+      </div>
+        <% if @value % @num_cols == 0 %>
+          </div>
+          <div class="row">
+        <% end %>
+        <% @value += 1 %>
+     <% end %>
+     </div>
+  </div>
+</div>
+<br>
+<% if admin_signed_in? %>
+  <%= link_to 'Edit Gallery', photos_url, class: "btn btn-primary"%>
+<% end %>
```

Comment:
Put this section inside one of the containers so that the button doesn't sit against the side of the page.

Reply:
fixed in new PR